### ISSUE TITLE
some coding is announced while using Gmail in internet explorer 11

### ISSUE THREAD

**Issue by YogeshChhabra** [nvaccess/nvda#7531]
1. Open internet explorer 11.
2. open inbox of your gmail account.
3. move to a link "create a filter" and navigate with down aero key.
4. NVDA will read some coding language.

### Expected behavior:
NVDA should not announce this coding language.

### Actual behavior:
NVDA is announcing some coding instead

### System configuration:
NVDA version:
NVDA 2017.3

NVDA Installed or portable:
Installed

Other information:

Windows version:
Microsoft Windows 7 [Version 6.1.7601]

Name and version of other software in use when reproducing the issue:
Using internet explorer 11

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by ehollig** [comment]
Confirmed. This appears to be a problem only using IE 11. The code is:
```
<script type="text/javascript" nonce="jvXFTV6ltoJDvjKbJ5mM41JfxWo">
var searchButtonElements=document.getElementsByClassName("search-form-submit");
for(i=0;i<searchButtonElements.length;i++)searchButtonElements[i].onclick=function(event){var urlParams={"s":"q","q":document.getElementById("sbq").value};urlParams[event.target.getAttribute("name")]=event.target.getAttribute("value");var encodedParams=[];for(var param in urlParams)encodedParams.push(encodeURIComponent(param)+"\x3d"+encodeURIComponent(urlParams[param]));document.getElementById("sbf").setAttribute("action","?"+encodedParams.join("\x26"))};
</script>
```

**Comment by YogeshChhabra** [comment]
what is the next step now to resolve this issue in the next version of NVDA? Is it enough from my side?

**Comment by Adriani90** [comment]
@YogeshChhabra, @ehollig  I cannot reproduce this issue in NVDA 2019.1.1. Are you still seeing this issue?

**Comment by ehollig** [comment]
Nope. No longer an issue. IE11 now reports, "Frame javascript" but is not able to read the content in this frame.
